ansible/group_vars/all.yml

```diff
@@ -19,12 +19,17 @@ default_users:
   authorized_keys:
   - files/keys/kzien.pub
 
+php_cli_version: 7.2
+php_cli_xdebug: false
+
 php_packages:
-- php7.0-opcache
-- php7.0-dom
-- php7.0-mbstring
+- php7.2-opcache
+- php7.2-dom
+- php7.2-mbstring
+
+composer_php_version: 7.2
```

Comment:
QUESTION: Does this relate directly to PHP version? Should it be separate from php_cli_version?

Reply:
Composer has a dependency on the php zip module. This ensures that it installs the right module as version number is a part of the package name. I think it would be better to default this to the CLI version number as that will always be what you are installing it against. I'll update this.